src/Autofac/IStartable.cs

```diff
@@ -12,10 +13,18 @@ namespace Autofac
     /// an <c>OnActivated()</c> event handler instead.
     /// </summary>
     /// <remarks>
+    /// <para>
     /// For equivalent "Stop" functionality, implement <see cref="IDisposable"/>. Autofac
     /// will always dispose a component before any of its dependencies (except in the presence
     /// of circular dependencies, in which case the components in the cycle are disposed in
     /// reverse-construction order.)
+    /// </para>
+    ///
```

Comment:
```suggestion
```

It's more like HTML where you don't actually put spaces between paragraphs.

Reply:
I mean, to be honest, I still put spaces in my HTML...